src/view/components/splash/index.scss

```diff
@@ -0,0 +1,31 @@
+.container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1000;
+}
+
+.logo {
+  width: 200px;
+  height: auto;
+}
+
+.progress-bar {
+  width: 80%;
+  height: 20px;
+  background-color: #ddd;
+  border-radius: 10px;
+  margin: 20px 0;
+}
+
+.progress {
+  height: 100%;
+  background-color: #333;
+  border-radius: 10px;
+}

```

Comment:
follow nesting
that's one reason to use SCSS over plain CSS

Reply:
Thank you so much for guiding me 

i made the suggested changes 
please review them 
@meganindya